Makefile.work

```diff
@@ -123,10 +123,16 @@ ifeq ($(CONFIGURED_ARCH), $(COMPILE_HOST_ARCH))
 SLAVE_BASE_IMAGE = $(SLAVE_DIR)
 MULTIARCH_QEMU_ENVIRON = n
 else ifneq ($(CONFIGURED_ARCH),)
+ifneq ($(CROSS_BLDENV),)
+SLAVE_BASE_IMAGE = $(SLAVE_DIR)-march-$(CONFIGURED_ARCH)
```

Comment:
This line is duplicated in `ifneq` and `else` blocks. Could you write once?

Reply:
Will fix.